src/arb/arb.jl

```diff
@@ -135,6 +133,35 @@ function convert(::Type{Float64}, x::arb)
     return Float64(x)
 end
 
+@doc Markdown.doc"""
+    fmpz(x::arb)
+
+Return $x$ as an `fmpz` if it represents an unique integer, else throws an
+error.
+"""
+function fmpz(x::arb)
+  (b, n) = unique_integer(x)
+  b == 0 ? error("Argument must represent be an unique integer") : return n
```

Comment:
```suggestion
  !b ? error("Argument must represent be an unique integer") : return n
```
I think `b` is just a boolean.